recipes/open62541/all/test_package/CMakeLists.txt

```diff
@@ -4,31 +4,8 @@ project(test_package LANGUAGES C)
 find_package(open62541 REQUIRED CONFIG)
 find_package(Threads REQUIRED)
 
-find_package(Python3 REQUIRED)
-set(PYTHON_EXECUTABLE ${Python3_EXECUTABLE})
-
-# Output directory for Nodeset Compiler
-set(GENERATE_OUTPUT_DIR "${CMAKE_BINARY_DIR}/src_generated/")
-file(MAKE_DIRECTORY "${GENERATE_OUTPUT_DIR}")
-include_directories("${GENERATE_OUTPUT_DIR}")
-ua_generate_nodeset_and_datatypes(
```

Comment:
This check is a little more special
What does it use to "generate"? I'm assuming it generates files from the `csv` and `bsd` files? in that case - we need to identify how they are generated - and attempt to replicate this in the `test_package` as well (like we did recently for protobuf)